ReIndexOrchestrator/index.js

```diff
@@ -0,0 +1,44 @@
+const df = require('durable-functions');
+const moment = require('moment');
+
+module.exports = df.orchestrator(function* orchestratorFunctionGenerator(context) {
+  context.log('Starting ReIndexOrchestrator');
+  const input = context.df.getInput();
+
+  context.log({ input });
+
+  const indexNames = input.parameters.indexNames;
+  const apimApiName = input.parameters.apimApiName;
+  let indexDefinition = input.parameters.indexDefinition;
+
+  if (!indexDefinition) {
+    indexDefinition = yield context.df.callActivity('GetIndexDefinition', indexNames.active);
+  }
+  context.log(indexDefinition);
+
+  const indexerName = yield context.df.callActivity('ReIndex', { indexDefinition, indexNames });
+  context.log({ indexerName });
+
+  const polling = { interval: 60, units: 'seconds' };
+  const expiryTime = moment().add(polling.interval * 15, polling.units);
+
+  while (moment.utc(context.df.currentUtcDateTime).isBefore(expiryTime)) {
+    // Put wait at beginning of loop rather than end so that reindexing is
+    // underway before first checking status
+    const nextCheck = moment
+      .utc(context.df.currentUtcDateTime)
+      .add(polling.interval, polling.units);
+    yield context.df.createTimer(nextCheck.toDate());
+    const indexerStatus = yield context.df.callActivity('GetIndexerStatus', indexerName);
+    context.log({ indexerStatus });
```

Comment:
Could some additional information be added to increase the usefulness of this message within the logs please?